frontend/src/widgets/workspace-shell/ui/workspace-shell.module.css

```
@@ -0,0 +1,111 @@
+.shell {
+  --workspace-shell-padding: clamp(20px, 4vw, 36px);
+  width: 100%;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)